packages/appstore/src/components/wallet-header/wallet-header.tsx

```diff
@@ -0,0 +1,133 @@
+import { Icon, Text } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import classNames from 'classnames';
+import React from 'react';
+
+type TWalletHeaderCommon = {
+    balance?: string;
+};
+
+type TWalletHeaderDemo = TWalletHeaderCommon & {
+    jurisdiction: 'virtual';
```

Comment:
jurisdiction could be one of below types, `virtual` is not one of them. you can define another props (account_type : real | demo) to differentiate between virtual and real
![image](https://user-images.githubusercontent.com/82078941/233997586-52dbf87d-29af-4d26-95fa-50bfe0f8e746.png)

Reply:
@mahdiyeh-deriv ok, I'll add one more prop for `account_type` and delete 'virtual' from `jurisdiction`